sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -31,9 +31,9 @@ To update generated files for chat service, run the following command
 
 ### Code generation settings
 ``` yaml
-tag: package-chat-2021-04-05-preview6
+tag: package-chat-2021-09-07
```

Comment:
If the version/tag changed in this readme.md file, I think you should try to regenerate the clients with autorest to ensure consistency. If nothing changed, I'd challenge the necessity of this line change. :-)

Reply:
Already done, there were no API changes

Reply:
Thanks for the explanation.